solr_schema_init/create_isb_core_schema.py

```diff
@@ -83,11 +85,20 @@ def createCopyField(source, dest):
 createField(
     "producedBy_samplingSite_description_text", "text_general", True, True, None
 )
+createCopyField("producedBy_samplingSite_description", "producedBy_samplingSite_description_text")
```

Comment:
This was missing in the source, but I think had manually run it previously on mars.  Anyhow, it should be checked in.